spec/Cabin/Infotainment.vspec

```diff
@@ -89,6 +89,12 @@ Navigation.DestinationSet.Longitude:
   max: 180
   unit: degrees
   description: Longitude of destination in WGS 84 geodetic coordinates.
+  
+Navigation.Mute:
+  datatype: string
+  type: actuator
+  allowed: ['Muted', 'Alerts only', 'Unmuted']
```

Comment:
I would prefer if the [style guide](https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/allowed/#recommendation-on-string-values) is followed, i.e. use upper case and underscore. (Alternatively we can discuss the style guide at next meeting to see if we still consider the style guide as relevant)

i.e. something like:

`allowed: ['MUTED', 'ALERT_ONLY', 'UNMUTED']`

Reply:
@erikbosch Thanks for quick comment! I am sorry that It is the same comment with previous my PR :( I have applied your comment.